src/hpp/gepetto/viewer.py

```diff
@@ -66,6 +69,26 @@ def __init__ (self, problemSolver, viewerClient = None, collisionURDF = False, d
         if collisionURDF:
             self.toggleVisual(False)
         self.client.gui.addToGroup (self.displayName, self.sceneName)
+        # create velocity and acceleration arrows :
+        self.displayArrows = displayArrows
+        if displayArrows :
+          if self.robot.client.robot.getDimensionExtraConfigSpace() < 6:
```

Comment:
This is a good idea. It is however dependent on hpp-rbprm. Shouldn't this go into a derived class in folder src/hpp/gepetto/rbprm ?

Reply:
Done in https://github.com/humanoid-path-planner/hpp-gepetto-viewer/pull/25/commits/4922216c726e9376b64ab7aac00af32636281038